day1.scala

```diff
@@ -0,0 +1,26 @@
+// using scala 3.0.2
+
+package day1
+
+import scala.io.Source
+
+@main def part1(): Unit =
+  val input = Source.fromFile("input/day1").mkString
+  val answer = part1(input)
+  println(s"The solution is $answer")
+
+@main def part2(): Unit =
+  val input = Source.fromFile("input/day1").mkString
+  val answer = part2(input)
+  println(s"The solution is $answer")
+
+def part1(input: String): String = 
+  val depths = input.split('\n').iterator.map(_.toInt)
```

Comment:
the input could be defirection a Seq[Int]

Reply:
```suggestion
  val depths = input.linesIterator.map(_.toInt)
```